tests/integ/test_stored_procedure.py

```diff
@@ -1150,3 +1159,62 @@ def test_anonymous_stored_procedure(session):
     )
     assert add_sp._anonymous_sp_sql is not None
     assert add_sp(1, 2) == 3
+
+
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC,
+    reason="Subprocess calls are not allowed within stored procedures",
+)
+def test_add_requirements_unsupported(session, resources_path):
+    test_files = TestFiles(resources_path)
+
+    with patch.object(session, "_is_anaconda_terms_acknowledged", lambda: True):
+        session.add_requirements(test_files.test_unsupported_requirements_file)
+        # Once scikit-fuzzy is supported, this test will break; change the test to a different unsupported module
+        assert set(session.get_packages().keys()) == {
+            "matplotlib",
+            "pyyaml",
+            "snowflake-snowpark-python",
+            "scipy",
+            "numpy",
+        }
+
+    @sproc
+    def run_scikit_fuzzy(_: Session) -> str:
+        import numpy as np
+        import skfuzzy as fuzz
+        from skfuzzy import control as ctrl
+
+        # Create fuzzy variables
+        temperature = ctrl.Antecedent(np.arange(0, 101, 1), "temperature")
+        humidity = ctrl.Antecedent(np.arange(0, 101, 1), "humidity")
+        fan_speed = ctrl.Consequent(np.arange(0, 101, 1), "fan_speed")
+
+        # Define fuzzy membership functions
+        temperature["cold"] = fuzz.trimf(temperature.universe, [0, 0, 50])
+        temperature["warm"] = fuzz.trimf(temperature.universe, [0, 50, 100])
+        humidity["dry"] = fuzz.trimf(humidity.universe, [0, 0, 50])
+        humidity["moist"] = fuzz.trimf(humidity.universe, [0, 50, 100])
+        fan_speed["low"] = fuzz.trimf(fan_speed.universe, [0, 0, 50])
+        fan_speed["high"] = fuzz.trimf(fan_speed.universe, [0, 50, 100])
+
+        # Define fuzzy rules
+        rule1 = ctrl.Rule(temperature["cold"] & humidity["dry"], fan_speed["low"])
+        rule2 = ctrl.Rule(temperature["warm"] & humidity["moist"], fan_speed["high"])
+
+        # Create fuzzy control system
+        fan_ctrl = ctrl.ControlSystem([rule1, rule2])
+        fan_speed_ctrl = ctrl.ControlSystemSimulation(fan_ctrl)
+
+        # Set inputs
+        fan_speed_ctrl.input["temperature"] = 30
+        fan_speed_ctrl.input["humidity"] = 70
```

Comment:
I would suggest having a computationally minimal `sproc` for testing purpose but not too strong of an opinion. WDYT @sfc-gh-sfan